test-zeekoe.py

```diff
@@ -195,6 +196,12 @@ def pay_all(self):
     def close(self):
         close_channel(self.cust_config, self.channel_name, self.verbose)
 
+    def expire(self):
+        # TODO: Get channel_id from a channel_name
```

Comment:
Interesting. Is there already an open issue for this? If not, I think we should make one.

Reply:
created an issue for it, #291